site/content/docs/02-template-syntax.md

```diff
@@ -141,6 +141,13 @@ Text can also contain JavaScript expressions:
 <p>{a} + {b} = {a + b}.</p>
 ```
 
+---
+
+However, if you're using regular expression literal that starts with `/`, you'll need to wrap it in parentheses.
```

Comment:
```suggestion
However, if you're using a regular expression literal that starts with `/`, you'll need to wrap it in parentheses.
```

Reply:
Thanks! I've updated it with some additional changes as well